readme.md

```diff
@@ -725,3 +725,9 @@ What's the best tool for automatically ignoring long email threads about tech bu
 **A:** They retreated back into their shell
 
 ---
+
+**Q:** Which highway can Toronto developers never seem to find?
+
+**A:** The 401
```

Comment:
This would make more sense if it was 404 🤔 

Perhaps you could phrase the question to be something involving Authentication.

Reply:
There is a 404 highway in Ontario.